demo-java/app/src/main/res/layout/activity_main.xml

```diff
@@ -39,7 +39,7 @@
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
     android:layout_margin="@dimen/spacing_small"
-    android:text="@string/programmatic_autocomplete"/>
+    android:text="@string/programmatic_as_you_type_autocomplete"/>
```

Comment:
Suggestion (based on suggested string value in strings.xml): rename this to @string/programmatic_autocomplete_geocoding